ci.yml

```diff
@@ -14,67 +14,67 @@ variables:
 pool: windevbuildagents
 
 steps:
-- checkout: self
-  clean: false
+  - checkout: self
```

Comment:
nit: was the additional whitespace intentional?

Reply:
+1 would be nice if you could remove the indentation changes in this file to make the diff more accurate

Reply:
Looks like the linter forced those changes in white space formatting.